server/app/durablejobs/jobs/FixApplicantDobDataPathJob.java

```diff
@@ -0,0 +1,51 @@
+package durablejobs.jobs;
+
+import com.google.common.base.Preconditions;
+import durablejobs.DurableJob;
+import io.ebean.DB;
+import io.ebean.Database;
+import java.time.LocalDate;
+import java.util.List;
+import models.Applicant;
+import models.PersistedDurableJob;
+import services.WellKnownPaths;
+import services.applicant.ApplicantData;
+
+/**
+ * Finds applicant data objects with applicant_date_of_birth set to a number and changes it to a
+ * nest object.
+ */
+public final class FixApplicantDobDataPathJob extends DurableJob {
+  private final Database database;
+
+  private final PersistedDurableJob persistedDurableJob;
+
+  public FixApplicantDobDataPathJob(PersistedDurableJob persistedDurableJob) {
+    this.persistedDurableJob = Preconditions.checkNotNull(persistedDurableJob);
+    this.database = DB.getDefault();
+  }
+
+  @Override
+  public PersistedDurableJob getPersistedDurableJob() {
+    return persistedDurableJob;
+  }
+
+  @Override
+  public void run() {
+    List<Applicant> applicants = database.find(Applicant.class).findList();
```

Comment:
I feel like there must be a way to make this more performant, but I had a hard time figuring it out. I attempted to query only for applicants that have the `applicant_date_of_birth` path in the json object, but I ran into the same problem as Swathi because our "json" is stored as a string and I couldn't figure out how to write it in raw sql.

Reply:
See comment here on how to cast the string to jsonb inside the postgres query: https://github.com/civiform/civiform/blob/main/server/app/repository/ProgramRepository.java#L361-L366